quickstart/ia32-generic-qemu.md

```diff
@@ -107,6 +107,174 @@ The number of detected cores is presented during kernel initialization.
 
 ![Image](../_static/images/quickstart/qemu-ia32-generic-smp.png)
 
+## Network setup on ia32-generic-qemu
+
+```{note}
+This guide was tested on `Ubuntu 24.04 LTS` host OS.
+```
+
+There are few steps to follow:
+
+### 1. Create and set up `vibr0` bridge on a host using `qemu-bridge-helper`
+
+Install the required package and ensure that `libvirtd` is running:
+
+```console
+sudo apt update
+sudo apt install qemu-system-common libvirt-clients libvirt-daemon
+systemctl enable libvirtd.service
+systemctl start libvirtd.service
+```
+
+Start the default network bridge, and configure it to run on startup.
+
+```console
+sudo virsh net-autostart --network default
+sudo virsh net-start --network default
+```
+
+After that verify that the IP range `192.168.122.1/24` is reported by the `vibr0` bridge:
+
+```console
+ip addr show virbr0
+```
+
+The expected output:
+
+```console
+virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
+  link/ether xx:xx:xx:xx:xx:xx brd ff:ff:ff:ff:ff:ff
+  inet 192.168.122.1/24 brd 192.168.122.255 scope global virbr0
+    valid_lft forever preferred_lft forever
+```
+
+If `/etc/qemu` directory does not exist, create it and provide required privileges:
+
+```console
+sudo mkdir /etc/qemu
+sudo chmod 755 /etc/qemu
+```
+
+Provide bridge configuration for QEMU:
+
+```console
+echo "allow virbr0" | sudo tee -a /etc/qemu/bridge.conf > /dev/null
+sudo chmod 755 /etc/qemu/bridge.conf
```

Comment:
https://wiki.qemu.org/Features/HelperNetworking

```
The minimum required to run the default helper with the default bridge br0 is:

/etc/qemu/bridge.conf root:qemu 0640 
```

Definitely chown to root:root (don't have qemu group in ubuntu)
chmod probably to 644 like other files in /etc